diff --git a/scripts/package/builddeb b/scripts/package/builddeb
index 67cd420dc..87773cd20 100755
--- a/scripts/package/builddeb
+++ b/scripts/package/builddeb
@@ -146,6 +146,7 @@ if [ "$ARCH" = "um" ] ; then
 else
 	cp System.map "$tmpdir/boot/System.map-$version"
 	cp $KCONFIG_CONFIG "$tmpdir/boot/config-$version"
+	cp $srctree/arch/arm64/boot/Image "$tmpdir/boot/Image"
 fi
 cp "$($MAKE -s -f $srctree/Makefile image_name)" "$tmpdir/$installed_image_path"
 
@@ -208,6 +209,16 @@ EOF
 	chmod 755 "$tmpdir/DEBIAN/$script"
 done
 
+cat <<EOF > "$tmpdir/DEBIAN/postinst"
+#!/bin/sh
+set -e
+
+# Create symlink to dtbs at /boot
+mkdir -p /boot/dtb
+ln -sf /usr/lib/$packagename/ti /boot/dtb/ti
+exit 0
+EOF
+
 if [ "$ARCH" != "um" ]; then
 	if is_enabled CONFIG_MODULES; then
 		deploy_kernel_headers debian/linux-headers
diff --git a/scripts/package/buildtar b/scripts/package/buildtar
index cb54c7f1a..891d5e0f1 100755
--- a/scripts/package/buildtar
+++ b/scripts/package/buildtar
@@ -128,6 +128,7 @@ case "${ARCH}" in
 				break
 			fi
 		done
+		cp -v -- "${objtree}/arch/arm64/boot/Image" "${tmpdir}/boot/Image"
 		;;
 	riscv)
 		for i in Image.bz2 Image.gz Image; do
